operator/pkg/apis/istio/v1alpha1/validation/validation.go

```diff
@@ -216,6 +217,16 @@ func CheckAutoScaleAndReplicaCount(values *valuesv1alpha1.Values, spec *v1alpha1
 	return
 }
 
+const distrolessVariant = "distroless"
+
+func CheckTag(values *valuesv1alpha1.Values, spec *v1alpha1.IstioOperatorSpec) (errs util.Errors, warnings []string) {
+	tag := spec.GetTag().GetStringValue()
+	if values.GetGlobal().GetVariant() == distrolessVariant && strings.HasSuffix(tag, distrolessVariant) {
+		errs = util.AppendErr(errs, fmt.Errorf("values.global.variant is already 'distroless'; 'distroless' tag addition is unnecessary"))
```

Comment:
if the user has indicated in two different ways that they want `-distroless`, should we just give them that and move on, rather than giving them an error that says they can't have `-distroless-distroless`?  @howardjohn